editor/components/arrangement-toolbar/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import { first, last } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import Dashicon from 'components/dashicon';
+
+function BlockArrangement( { onMoveUp, onMoveDown, isFirst, isLast } ) {
+	return (
+		<div className="editor-arrangement-toolbar">
+			<button
+				className={ classnames( 'editor-arrangement-toolbar__control', { 'is-disabled': isFirst } ) }
```

Comment:
Line is a little long. If not as separate variables, at least some line breaks could help clear it up:

```js
className={ classnames( 'editor-arrangement-toolbar__control', {
	'is-disabled': isFirst
} ) }
```